modules/docs/mdx/13.0-UPGRADE-GUIDE.mdx

```diff
@@ -219,44 +281,43 @@ import {dubLogoBlue} from "@workday/canvas-kit-react/common"
 import {dubLogoPrimary} from "@workday/canvas-kit-preview-react/common"
 
 <div dangerouslySetInnerHTML={__html: dubLogoPrimary} />
-
-// In v13
+```
+**After in v13**
+```tsx
 // New logos have been promoted to Main common AND renamed
 import {dubLogoPrimary} from "@workday/canvas-kit-preview-react/common"
 
 <div dangerouslySetInnerHTML={__html: dubLogoPrimary} />
 ```
 
-🤖 The codemod will handle the change of `'dubLogoBlue', 'dubLogoWhite', 'wdayLogoBlue', 'wdayLogoWhite'` to
-`'dubLogoPrimary', 'dubLogoReversed', 'wdayLogoPrimary', 'wdayLogoReversed'` as shown above in the table. It will also handle
-updating your imports.
-
-From:
-```tsx
-import {dubLogoBlue} from "@workday/canvas-kit-react/common"
-```
-To:
-```tsx
-import {dubLogoPrimary} from "@workday/canvas-kit-react/common"
-```
+> 🤖 The codemod will handle the change of `'dubLogoBlue', 'dubLogoWhite', 'wdayLogoBlue', 'wdayLogoWhite'` to
+> `'dubLogoPrimary', 'dubLogoReversed', 'wdayLogoPrimary', 'wdayLogoReversed'` as shown above in the table. It will also handle
+> updating your imports.
 
 
 ## Infrastructure
 
-### Supporting `react-jsx`
+### Supporting `react-jsx` 🚨
 
 We've updated the `jsx` flag in our `tsconfig` to `react-jsx`. As part of this change, we've also updated our peer dependencies for our packages to a minimum version of `react@17.0.0`. This change is to provide support for modern technologies like `vite` and ES modules.
 
 ##### Breaking Changes
 
 If you're using `react@16.x.x`, you'll need to upgrade to `react@17.x.x` to use Canvas Kit v13.
 
->**Note:** You'll need to update the way `jsx` transpiles. The automatic runtime feature, enabled through the `@babel/preset-react` preset with the `runtime: "automatic"` option, handles the importing of functions that JSX transpiles to.
+> **Note:** You'll need to update the way `jsx` transpiles. The automatic runtime feature, enabled through the `@babel/preset-react` preset with the `runtime: "automatic"` option, handles the importing of functions that JSX transpiles to.
 
-```sh
 
 ## Troubleshooting
 
+- When upgrading to the latest major version of Canvas Kit, all related Canvas Kit packages should be updated at the same time:
+	- `"@workday/canvas-kit-docs": "^12.5.6"` -> `"@workday/canvas-kit-docs": "^13.0.0"`
```

Comment:
Question: Is anyone using the Canvas Kit docs package outside of our team?

Reply:
Good point, removed it.